internal/cache/cache.go

```diff
@@ -197,61 +197,51 @@ func (c *LoadingCache) Close() {
 	close(c.done)
 }
 
-// keysWithTs includes list of keys with ts. This is for sorting keys
-// in order to provide least recently added sorting for size-based eviction
-type keysWithTs []struct {
-	key string
-	ts  time.Time
+// removeOldest removes the oldest item from the cache. Has to be called with lock!
+func (c *LoadingCache) removeOldest() {
```

Comment:
this name is misleading. As far as I undesrstand it is oldes for TTL case, but not in LRU case

Reply:
Oldest by creation time in case of LRC, oldest by access time in case of LRU.

Reply:
well, "oldest" means an age, at least this is how I read it. In the case of LRU this is not "age" but rather "access time". Not sure what the right name here, but "oldest" is misleading for real